SCAutolib/models/file.py

```diff
@@ -144,6 +197,23 @@ def clean(self):
         except FileNotFoundError:
             logger.info(f"{self._conf_file} does not exist. Nothing to do.")
 
+    def backup(self):
```

Comment:
Backup and restore methods are not compatible with SSSDConf class  which create backups in the create method and restores original files in the clean mathod. Also different names (_backup_original = Path(LIB_BACKUP, 'original_sssd.conf')) are used in SSSDConf class. Current implementation does not reflect it. 

We should either 1) keep current implementation of SSSDConf class and overwrite backup and clean method in SSSDConf class to preven unexpected behavior 

or 2) modify create method of SSSDConf class to use parent class backup method (to lower deduplication of code) and rename SSSDConf class clean method to restore so it overwrites parent restore method. 

I guess there can also be different solution. We just need to make sure it's consistent.

Reply:
Yes, makes sense. I would prefer to go in a second way.